pdsClient/pdsClient_test.go

```diff
@@ -73,6 +73,96 @@ func MakeClientWriterForRecordType(pdsUser E3dbPDSClient, clientID string, recor
 
 var validPDSEncryptedAccessKey, _ = MakeClientWriterForRecordType(validPDSUser, validPDSUserID, defaultPDSUserRecordType)
 
+func TestBatchGetRecordsReturnsBatchofUserRecords(t *testing.T) {
```

Comment:
Seeing happy path testing, but no sad path testing. Would like to see a couple sad path tests to ensure failures happen as we expect.

Reply:
added some more unhappy path testing.